vic/drivers/classic/src/get_global_param.c

```diff
@@ -427,12 +384,14 @@ get_global_param(FILE *gp)
                     options.RC_MODE = RC_JARVIS;
                 }
             }
+
```

Comment:
Are these whitespace changes from `uncrustify` (see following as well). If so - then fine. If not, then don't commit whitespace only changes.